include/seqan/graph_algorithms/depth_first_search.h

```diff
@@ -30,13 +30,17 @@
 //
 // ==========================================================================
 // Author: Tobias Raussch <rausch@embl.de>
+// Author: Ryan Wick <rrwick@gmail.com>
 // ==========================================================================
 // Implementation of Depth-First-Search algorithm.
 // ==========================================================================
 
 #ifndef INCLUDE_SEQAN_GRAPH_ALGORITHMS_DEPTH_FIRST_SEARCH_H_
 #define INCLUDE_SEQAN_GRAPH_ALGORITHMS_DEPTH_FIRST_SEARCH_H_
 
+#include <vector>
+#include <utility>
```

Comment:
Can you please move these includes to the umbrella header: in include/seqan/graph_algorithms.h

Here: https://github.com/seqan/seqan/blob/master/include/seqan/graph_algorithms.h#L39